book/_quarto.yml

```diff
@@ -41,16 +44,19 @@ execute:
   echo: true
 
 engine: knitr
+
 webr:
-  channel-type: "post-message"
-  packages: ["tern", "random.cdisc.data", "dplyr"]
-  show-startup-message: false
-  autoload-packages: false
+  packages:
+    - tern
+    - random.cdisc.data
```

Comment:
should we also include pharamverseadam

Reply:
Currently `pharmaverseadam` is not used at all. If we want to change this then that probably had to be done as a separate PR not to explode with this one